src/main/java/com/internet/shop/filter/IpFilter.java

```diff
@@ -0,0 +1,42 @@
+package com.internet.shop.filter;
+
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class IpFilter implements Filter {
```

Comment:
this filter will only confuse you while testing other filters(cause your are testing only locally). It's good that you have figured how to do that, but let's remove it.